docs/analyzers/DisposedBeforeAsyncRunAnalyzer.md

```diff
@@ -0,0 +1,32 @@
+---
+title: DisposedBeforeAsyncRunAnalyzer
+category: analyzers
+categoryindex: 1
+index: 11
+---
+
+# DisposedBeforeAsyncRunAnalyzer
+
+## Problem
+
+A `use` statement in an `async` or `task` returning function can be problematic as the value is disposed before the returned workflow is run.
+
+```fsharp
+let f () =
+    use t = new DisposableThing()
+    async {
+        return "hi"
+    }
+```
+
+## Fix
+
+If this is not an issue for your context, you can disable the warning with a comment on top of the `use` statement.
```

Comment:
I think we probably want to highlight that `t` should probably go in the `async` instead.
And then in another code example show that we can also disable this via comment.